js/main.js

```diff
@@ -31,3 +30,56 @@ function handleSubmit(event) {
 }
 
 $form.addEventListener('submit', handleSubmit);
+
+function createEntry(entry) {
+  var li = document.createElement('li');
+  var row = document.createElement('div');
+  row.className = ('row');
+  li.appendChild(row);
+
+  var half = document.createElement('div');
+  half.className = ('column-half');
+  row.append(half);
+
+  var img = document.createElement('img');
+  img.setAttribute('src', entry.url);
+  half.appendChild(img);
+
+  var otherHalf = document.createElement('div');
+  otherHalf.className = ('column-half');
+  row.append(otherHalf);
+
+  var title = document.createElement('h3');
+  title.textContent = entry.title;
+  otherHalf.appendChild(title);
+
+  var p = document.createElement('p');
+  p.textContent = entry.notes;
+  otherHalf.appendChild(p);
+
+  return li;
+}
+window.addEventListener('DOMContentLoaded', function () {
+  for (var i = 0; i < data.entries.length; i++) {
+    var newLi = createEntry(data.entries[i]);
+
+    $ul.appendChild(newLi);
+  }
+});
+
+var $view = document.querySelectorAll('.view');
+
+function handleContainer(event) {
+  if (!event.target.matches('.click')) {
+    return;
+  }
+  for (var i = 0; i < $view.length; i++) {
+    var views = $view[i];
+    if (views.getAttribute('data-view') === event.target.getAttribute('data-view')) {
+      views.className = 'view';
+    } else {
+      views.className = 'view hidden';
+    }
+  }
+}
```

Comment:
You shouldn't make this function reliant on an event object. You want this to have a parameter named something like `view` which is a string of the name of the view to go to. That way, you can use this function to switch between views in any context, not just an event handler context 🤓